.github/workflows/deploy-master.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Master
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.12.12
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
```

Comment:
While I added `buildpack` to other actions, I wonder if we actually need it 🤔 So for this one, I didn't add one to see if it still works correctly :)